Documentation/ceph-csi-drivers.md

```diff
@@ -129,9 +129,6 @@ kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/storegeclass.
 
 ## Create RBD Secret
```

Comment:
Is there any reason the operator can't create this secret automatically? This is a tedious setup step that seems like it should be automated. The operator can create the secret just like it creates the configmap for mons in the cluster, right?

Reply:
the user can be the admin or any other user if the user chose to create a separate user who can access the pool created after creation cluster.

to do it in the code we can create a secret for admin user and as we don't know the pool name we cannot create a new user and add permission to the poll and create a secret for the same user.

note: the secret name and namespace has to  be mentioned in the storage class

Reply:
One of the next improvements post the current patch by @phlogistonjohn to create a CSI instance per rook operator (https://github.com/rook/rook/pull/3373), was to generate the secrets when an rbd pool or CephFS is created, for use by the StorageClass. IOW, create the secret by the operator for use by the StorageClass as you state @travisn. I would say that is the next logical step for hands off automation for users. @phlogistonjohn any comments/thoughts?

Reply:
@ShyamsundarR are we going to create a new user for a created pool or we will create secret from admin user?

Reply:
We would be creating a new user and using that in the created secret, that the user can use in the StorageClass. At least this is the initial thought on the subject.

Reply:
then are we going to create storage class also? 
what if the user doesn't want to create a new user just want to use admin user?

Reply:
@Madhu-1 We possibly want to discuss this in the PR/issue (which is possibly in the future anyway) that adds this change instead of here?

Reply:
sure let us open an issue and continue our discussion on it

Reply:
Yes I just mentioned this in #3373 too. I will file the issue now.

Reply:
(done)